### DISCUSSION TITLE
is there a replication solution for docker hosting?

### DISCUSSION THREAD

**Discussion by Ansellmo** [outline/outline#7947] [Self Hosting]
I already have a server where Outline is installed via Docker. And for a high availability solution, I would like to know if there is a way to replicate data between two Docker installations?

**Comment by tommoor** [comment]
>  I would like to know if there is a way to replicate data between two Docker installations?

No, there is no such thing. You can simply run more instances of most of the services:
https://github.com/outline/outline/blob/main/docs/SERVICES.md
